share/doc/homebrew/Tips-N'-Tricks.md

```diff
@@ -136,3 +136,9 @@ inline patches in Vim.
 [homebrew-mode](https://github.com/dunn/homebrew-mode) provides syntax
 highlighting for inline patches as well as a number of helper functions
 for editing formula files.
+
+### Atom
+
+[language-hombrew-formula](https://atom.io/packages/language-hombrew-formula)
+adds highlighting and diff support (with the
```

Comment:
Nope. This is correct as it matches the sentence. It's likely the diff doesn't make this obvious.